cmd/fortifyExecuteScan.go

```diff
@@ -400,19 +402,31 @@ func checkArtifactStatus(config fortifyExecuteScanOptions, sys fortify.System, p
 	return nil
 }
 
-func verifyScanResultsFinishedUploading(config fortifyExecuteScanOptions, sys fortify.System, projectVersionID int64, buildLabel string, filterSet *models.FilterSet, numInvokes int) error {
+func verifyScanResultsFinishedUploading(config fortifyExecuteScanOptions, sys fortify.System, projectVersionID int64, buildLabel string, filterSet *models.FilterSet) error {
+	const pollingDelay = 10 * time.Second
+	var timeout = time.Duration(config.PollingMinutes) * time.Minute
+	return verifyScanResultsFinishedUploadingWithDurations(config, sys, projectVersionID, buildLabel, filterSet, pollingDelay, timeout)
+}
+
+func verifyScanResultsFinishedUploadingWithDurations(config fortifyExecuteScanOptions, sys fortify.System, projectVersionID int64, buildLabel string, filterSet *models.FilterSet, pollingDelay, timeout time.Duration) error {
```

Comment:
The observation is correct, but it is also passed to `checkArtifactStatus`, where it again enriches error messages. I didn't want to refactor too much all at once. Since I wanted to actually unit-test the polling, I needed to make the timeouts configurable, otherwise the tests would run at least for a minute. But I didn't want to refactor beyond that concern.